src/plugins/abrt-action-generate-backtrace.c

```diff
@@ -78,7 +78,10 @@ int main(int argc, char **argv)
         debuginfo_dirs = libreport_xasprintf("%s:%s", debuginfo_location, i_opt);
 
     /* Create gdb backtrace */
-    char *backtrace = abrt_get_backtrace(dump_dir_name, exec_timeout_sec,
+    struct dump_dir *dd = dd_opendir(dump_dir_name, /*flags:*/ 0);
+    if (!dd)
+        return 1;
```

Comment:
Would it make sense to log an error here?

Reply:
Not really, IMO. `dd_opendir` and functions called therefrom already do that.

Reply:
Right. It's enough if `dd_opendir` complains in my opinion.